src/shipchain_common/rpc.py

```diff
@@ -46,7 +46,9 @@ def call(self, method, args=None):
         try:
             with TimingMetric('engine_rpc.call', tags={'method': method}) as timer:
                 response_json = settings.REQUESTS_SESSION.post(self.url,
-                                                               data=json.dumps(self.payload, cls=DecimalEncoder)).json()
+                                                               data=json.dumps(self.payload, cls=DecimalEncoder),
+                                                               timeout=getattr(settings, 'REQUESTS_TIMEOUT', 4.5)
```

Comment:
Do I need to add `settings.REQUESTS_TIMEOUT`  param in profiles and transmission branches related to the integration to this package?

Reply:
No, the second parameter to getattr is a default of 270 seconds :+1: